README.md

```diff
@@ -40,7 +41,14 @@ modalForms: Ember.inject.service('modal-forms'),
 isModalActive: Ember.computed.readOnly('modalForms.isModalActive'),
 ```
 
-Change the active state of the modal in your component
+### Modal component
+Inject the [remodal](http://sethbrasile.github.io/ember-remodal/#/styling) and modal-forms services in your component
+```js
+remodal: inject.service(),
+modalForms: inject.service('modal-forms'),
```

Comment:
Ollpoll k op mpll I'll lll look o l I'll illllllklllllol
On May 20, 2016 9:22 AM, "Steven Glanzer" <notifications@github.com> wrote:

> In README.md
> <https://github.com/ciena-frost/ember-frost-modal-input/pull/23#discussion_r64068390>
> :
>
> > @@ -40,7 +41,14 @@ modalForms: Ember.inject.service('modal-forms'),
> >  isModalActive: Ember.computed.readOnly('modalForms.isModalActive'),
> >  ```
> >
> > -Change the active state of the modal in your component
> > +### Modal component
> > +Inject the [remodal](http://sethbrasile.github.io/ember-remodal/#/styling) and modal-forms services in your component
> > +```js
> > +remodal: inject.service(),
> > +modalForms: inject.service('modal-forms'),
>
> Unfortunately I don't believe the overlay is capable of a blur - the blur
> must be applied to the root application element in the DOM
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ciena-frost/ember-frost-modal-input/pull/23/files/4379d33380e518302998bbfb6f2f418eb0de47b3#r64068390>
>